tools/StaticAnalysis/SerializedCmdlets/Microsoft.Azure.Commands.Compute.dll.json

```diff
@@ -30873,7 +30873,7 @@
           "ValidateNotNullOrEmpty": true
```

Comment:
If you add the alias, there would be no breaking change here.  If it turns out you do need a breaking change, please add a suppression here:  https://github.com/Azure/azure-powershell/blob/preview/tools/StaticAnalysis/Exceptions/BreakingChangeIssues.csv 

Rather than updating this json file.